huntsman/dome/musca.py

```diff
@@ -123,15 +123,24 @@ def open(self):
         return False
 
     def close(self):
-        """Short summary.
+        """Close the shutter using musca.
 
         Returns
         -------
-        type
-            Description of returned object.
-
+        Boolean
+            True if Closed, False if it did not Close.
         """
-        self.logger.warning('musca Close function not yet operational')
+        if self.is_closed():
+            return True
+
+        self._write_musca(Protocol.CLOSE_DOME, 'Closing dome shutter')
+        time.sleep(HuntsmanDome.MOVE_TIMEOUT)
+
+        v = self.status()[Protocol.SHUTTER]
+        if v == Protocol.CLOSE:
+            return True
+        self.logger.warning('HuntsmanDome.open wrong final state: {!r}', v)
```

Comment:
Seems like we'll eventually need an email or twitter service to alert us here. :)